saleor/graphql/product/tests/test_product.py

```diff
@@ -9705,12 +9706,14 @@ def test_categories_query_with_filter(
         name="Category1",
         slug="slug_category1",
         description=dummy_editorjs("Description cat1."),
+        description_plaintext="Description cat1.",
```

Comment:
Search uses ilike, so we shouldn't confirm that it will find a part of the text from description_plaintext?
Like search `"Description cat1."` will also work for `description_plaintext="Sumpoer important Description cat1.",`